amigo_api.py

```diff
@@ -66,7 +81,4 @@ def send_analytics_event(self, category, action, label):
             ejson = json.dumps(e)
             e64 = base64.b64encode(bytes(ejson, 'utf-8'))
             url = 'http://api.mixpanel.com/track/?data=' + str(e64)
-            try:
```

Comment:
I never touched anything from send_analytics_event. no idea why these lines have been removed but if they were important I will add them again.

Reply:
You are correct, those lines should be there.